lib/config.rb

```diff
@@ -26,7 +27,7 @@ def url
     end
 
     def builds_dir
-      @builds_path ||= File.join($root_path, 'tmp', 'builds')
+      @builds_path ||= File.join($root_path, 'tmp', 'builds', 'runner-' + @runner.to_s)
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.